src/Maatwebsite/Excel/Readers/ChunkedReadJob.php

```diff
@@ -64,7 +69,8 @@ public function __construct(
         $startIndex,
         $chunkSize,
         callable $callback,
-        $shouldQueue = true
+        $shouldQueue = true,
+        $encoding = null
```

Comment:
Can you update the docblock as well please

Reply:
Yes, of course. I have updated the docblock.